osl/interface_linux.go

```diff
@@ -17,15 +17,16 @@ import (
 type IfaceOption func(i *nwIface)
 
 type nwIface struct {
-	srcName     string
-	dstName     string
-	master      string
-	dstMaster   string
-	address     *net.IPNet
-	addressIPv6 *net.IPNet
-	routes      []*net.IPNet
-	bridge      bool
-	ns          *networkNamespace
+	srcName        string
+	dstName        string
+	master         string
+	dstMaster      string
+	address        *net.IPNet
+	addressIPv6    *net.IPNet
+	routes         []*net.IPNet
+	bridge         bool
+	ns             *networkNamespace
+	addressAliases []*net.IPNet
```

Comment:
Minor.
I would call them `secondaryAddresses`, it seems the common terminology. The fact a driver can use them as alias address it is a specific use case, I think, which should not dictate the name.